scripts/test-ssl-death-alert.py

```diff
@@ -0,0 +1,73 @@
+# CVE-2016-8610
+# SSL Death Alert
+# OpenSSL SSL/TLS SSL3_AL_WARNING undefined alert flood remote DoS
+
+from __future__ import print_function
+import traceback
+import sys
+
+from tlsfuzzer.runner import Runner
+from tlsfuzzer.messages import Connect, ClientHelloGenerator, \
+        ClientKeyExchangeGenerator, ChangeCipherSpecGenerator, \
+        FinishedGenerator, AlertGenerator
+from tlsfuzzer.expect import ExpectServerHello, ExpectCertificate, \
+        ExpectServerHelloDone, ExpectChangeCipherSpec, ExpectFinished
+from tlslite.constants import CipherSuite, AlertLevel, AlertDescription
+
+
+NUMBER_OF_ALERTS = 4
+
+
+def main():
+    conversations = {}
+
+    conversation = Connect("localhost", 4433, version=(3, 3))
+    node = conversation
+    ciphers = [CipherSuite.TLS_RSA_WITH_AES_128_CBC_SHA]
+    node = node.add_child(ClientHelloGenerator(ciphers))
+    for i in range(0, NUMBER_OF_ALERTS):  # sending alerts during handshake
+        node = node.add_child(AlertGenerator(  # alert description: 46, 41, 43
+            AlertLevel.warning, AlertDescription.unsupported_certificate))
+    node = node.add_child(ExpectServerHello())
+    node = node.add_child(ExpectCertificate())
+    node = node.add_child(ExpectServerHelloDone())
+    node = node.add_child(ClientKeyExchangeGenerator())
+    node = node.add_child(ChangeCipherSpecGenerator())
+    node = node.add_child(FinishedGenerator())
+    node = node.add_child(ExpectChangeCipherSpec())
+    node = node.add_child(ExpectFinished())
+
+    conversations["SSL Death Alert"] = conversation
```

Comment:
About second test - I will send one more alert and expect only one alert from server after `ExpectServerHelloDone()`, right? But what next, after expecting that alert? Should test end after receiving that alert or should it try finish handshake in normal order (client key exchange, change cipher spec, etc.)?